extensions/exposition/source/directives/octets/Context.ts

```diff
@@ -8,7 +8,8 @@ export class Context extends Directive {
 
   public constructor (value: unknown) {
     super()
-    schemas.context.validate(value)
+
+    assert.ok(typeof value === 'string', 'Incorrect \'octets:context\' format')
```

Comment:
NIT: Can we improve error message here? 

```suggestion

    assert.ok(typeof value === 'string', `Incorrect 'octets:context' format (expected 'string', but got '${typeof value}'`)
```

Reply:
https://github.com/toa-io/toa/pull/692/commits/e9171b162808d152f9a8bf2f504c9e31a61f09dc